### ISSUE TITLE
pip installation gets stuck

### ISSUE THREAD

**Issue by dokato** [SridharJagannathan/pyroglancer#6]
I tried to remove the dependencies that it hangs on, but it's hopeless effort as it keeps reoccurring in others.

```
 resolver with stricter constraints to reduce runtime. If you want to abort this run, you can press Ctrl + C to do so. To improve how pip performs, tell us what happened here: https://pip.pypa.io/surveys/backtracking
  Using cached pyOpenSSL-16.0.0-py2.py3-none-any.whl (45 kB)
  Using cached pyOpenSSL-0.15.1-py2.py3-none-any.whl (102 kB)
  Using cached pyOpenSSL-0.15-py2.py3-none-any.whl (102 kB)
  Using cached pyOpenSSL-0.14.tar.gz (128 kB)

```

**Comment by SridharJagannathan** [comment]
Can you please try this and let me know what happens:

```
pip install pip==20.3
pip install git+git://github.com/SridharJagannathan/pyroglancer@master --use-deprecated=legacy-resolver
```

**Comment by SridharJagannathan** [comment]
Perhaps the issue in install failing to resolve dependencies is here: https://github.com/pypa/pip/issues/9215

**Comment by dokato** [comment]
Yeap that went through with this legacy resolver. As a sidenot: open3d is not yet available for python 3.9, I had to downgrade my version.
